services/knack_street_seg_updater.py

```diff
@@ -144,10 +144,14 @@ def main(args):
         features = query_atx_street(
             street_segment[config["primary_key"]], token
         )
-        time.sleep(1) # wait one second between requests to prevent being rate limited
```

Comment:
does this not make much of a difference?

Reply:
I tried higher than one second and it doesn't 🤷‍♂️. Seems to be random when AGOL likes to send me an error or a timeout.

Reply:
We can also just drop these changes since Prefect has the ability to retry and we won't usually have very many segments to update so this will be unlikely to happen in production.